hokusai/staging.yml

```diff
@@ -159,6 +159,48 @@ spec:
     - 104.16.0.0/12
     - 172.64.0.0/13
     - 131.0.72.0/22
+
+---
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: horizon
+    component: web
+    layer: application
+  name: horizon-web-internal
+  namespace: default
+spec:
+  ports:
+    - port: 8080
```

Comment:
Thinking it's better to create a new internal-only service rather than adding another listener for port 8080 to the external LoadBalancer and mapping to a NodePort, which will then likely suffer from the issues raised in https://github.com/artsy/doppler/pull/183 when we try and downgrade it.  Also probably not a great idea to circumvent the Nginx sidecar!